### PR TITLE
Incorrect result of Minimum op if u8/16/64 data type is used

### PR DESCRIPTION
**Pull request by egorshul**
### Details:
 - *Fixed Minimum op if u16 data type is used*

### Tickets:
 - *46762*

### CODE REVIEW

File: ngraph/test/backend/minimum.in.cpp [hunk 1]
```diff
@@ -79,25 +79,18 @@ NGRAPH_TEST(${BACKEND_NAME}, minimum_int64)
     test_case.run();
 }
 
-// TODO Refactor to use TestCase if u16 will be handled correctly
 NGRAPH_TEST(${BACKEND_NAME}, minimum_u16)
```
Comment (ilyachur) [inline ngraph/test/backend/minimum.in.cpp:82]: Can we use template reference tests?
And add new SLT for IE plugins?

**Review by jane-intel — APPROVED** [review]

**Review by GlebKazantaev — APPROVED** [review]

File: ngraph/test/backend/minimum.in.cpp (hunk 1 above)
Comment (ilyachur) [reply ngraph/test/backend/minimum.in.cpp:82]: You also didn't check u8 and u64 data types

Comment (egorshul) [reply ngraph/test/backend/minimum.in.cpp:82]: Done

Comment (ilyachur) [reply ngraph/test/backend/minimum.in.cpp:82]: @egorshul  Please remove these backend tests. And use reference tests instead.

Comment (egorshul) [reply ngraph/test/backend/minimum.in.cpp:82]: @ilyachur Reference tests do not work with unsigned type. "Interpreter backend doesn't implement evaluate method for OP Minimum".

Comment (ilyachur) [reply ngraph/test/backend/minimum.in.cpp:82]: I think in this case we need to extend evaluate method or add to evaluate_map such data types.

Comment (egorshul) [reply ngraph/test/backend/minimum.in.cpp]: @ilyachur Done

**Review by ilyachur — APPROVED** [review]

**Comment by ilyachur** [comment]
@egorshul Please resolve merge conflicts

File: ngraph/test/util/engine/ie_engines.cpp [hunk 2]
```diff
@@ -87,12 +170,6 @@ namespace
                                                   const size_t tolerance_bits)
     {
         const auto& computed_precision = computed->getTensorDesc().getPrecision();
-        const auto& expected_precision = expected->getTensorDesc().getPrecision();
-
-        if (computed_precision != expected_precision)
```
Comment (ilyachur) [inline ngraph/test/util/engine/ie_engines.cpp:92]: Why did you remove this check?

Comment (egorshul) [reply ngraph/test/util/engine/ie_engines.cpp:92]: When we work with a data type that converts to another, for example U16, we get computed_precision = I32 and expected_precision = U16.
